web/src/main/resources/webapp/template/view/documents/details.html

```diff
@@ -1,18 +1,121 @@
-        </form>
-    </section>
+	<section class="content-wrapper">
+		<section class="content-header">
+			<h1>Document <small th:text="${document == null} ? null : ${document.body.title}"></small></h1>
+		</section>
+		<section class="content">
+			<div class="row">
+				<div class="col-xs-12">
+					<th:block th:replace="_fragment/message :: message(${flash})"/>
+				</div>
+			</div>
+			<div class="row">
+				<div class="col-md-9">
+					<div class="box">
+						<div class="box-header">
+							<h3 class="box-title"></h3>
+						</div>
+						<div class="box-body">
+							<div class="form-group">
+								<label>Title</label>
+								<p class="form-control-static"
+								   th:text="${document == null} ? null : ${document.body.title}"></p>
+							</div>
+							<div class="form-group">
+								<label>Link</label>
+								<p class="form-control-static"
+								   th:text="${document == null} ? null : ${document.body.link}"></p>
+							</div>
+							<div class="calmdown cd-theme-default"
+								 th:text="${document == null} ? null : ${document.body.rawContent}"></div>
+						</div>
+					</div>
+				</div>
+				<div class="col-md-3">
+					<div class="box">
+						<div class="box-header">
+							<h3 class="box-title">Attributes</h3>
+						</div>
+						<div class="box-body">
+							<div class="form-group">
+								<p class="form-control-static"
+								   th:if="${document != null and document.body.enabled}">
+									<i class="fa fa-dot-circle-o text-green"></i> Enabled
+								</p>
+								<p class="form-control-static"
+								   th:if="${document == null or !document.body.enabled}">
+									<i class="fa fa-circle-o text-muted"></i> Disabled
+								</p>
+							</div>
+							<div class="form-group">
+								<label>Language</label>
+								<p class="form-control-static"
+								   th:text="${document != null and locale == document.body.locale}"></p>
+							</div>
+						</div>
+					</div>
+					<div class="box">
+						<div class="box-header">
+							<h3 class="box-title">SEO Attributes</h3>
+						</div>
+						<div class="box-body">
+							<div class="form-group">
+								<label>SEO Title</label>
+								<p class="form-control-static"
+								   th:text="${document == null} ? null : ${document.seo.metaTitle}"></p>
+							</div>
+							<div class="form-group">
+								<label>SEO Description</label>
+								<p class="form-control-static"
+								   th:text="${document == null} ? null : ${document.seo.metaDescription}"></p>
+							</div>
+							<div class="form-group">
+								<label>SEO Keywords</label>
+								<p class="form-control-static"
+								   th:text="${document == null} ? null : ${document.seo.metaKeywords}"></p>
+							</div>
+						</div>
+					</div>
+				</div>
+			</div>
+			<div class="row">
+				<div class="col-md-12">
+					<div class="box box-solid">
+						<div class="box-footer">
+							<a th:href="@{/documents/edit/{id}(id=__${#request.getAttribute('id')}__)}"
```

Comment:
Are you sure request.getAttribute call is working? There is a pretty new restriction in Thymeleaf since v3.0.8 (or so), which does not allow some of the request calls in links (had such an issue with pagination, see changeset of LFLT-315).

Reply:
It works, I tested it. On the other hand document.id is not available on the page. I try an other workaround.

Reply:
Guess you can leave it that way, let me do a quick search on the issue.